### PR TITLE
[TH] Bancho

### PR DESCRIPTION
**Pull request by C0NS03L**

### CODE REVIEW

**Review by DatPanDat — DISMISSED** [review]
Seems good.

File: wiki/Bancho/th.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Bancho (เซิฟเวอร์แชท)
+
```
Comment (TicClick) [inline wiki/Bancho/th.md:2]: the original article has tags and other metadata, please copy it over and, if applicable, translate the tags

File: wiki/Bancho/th.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+*อย่าจำสลับกันกับ [BanchoBot](/wiki/BanchoBot)*
+
+**Bancho** เป็นเซิฟเวอร์ [Internet Relay Chat](/wiki/Internet_Relay_Chat) (*IRC*) สำหรับ osu! การลงชื่อเข้าใช้ในเกมด้วยบัญชี osu! จะเชื่อมต่อเข้ากับ Bancho โดยอัตโนมัติ
```
Comment (TicClick) [inline wiki/Bancho/th.md:5]: you missed the full stop

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Bancho/th.md (hunk 2 above)
Comment (C0NS03L) [reply wiki/Bancho/th.md:5]: There is actually no full stop in Thai.

File: wiki/Bancho/th.md (hunk 1 above)
Comment (C0NS03L) [reply wiki/Bancho/th.md:9]: Added the tags

**Review by TicClick — APPROVED** [review]

**Pull request merged**